recipes/retry/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "retry" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8bfa8b99b69c4506d6f5bd3b0aabf77f98cdb17f3c9fc3f5ca820033336fba4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pbr
+  run:
+    - python
+    - decorator>=3.4.2
```

Please add a space between package name and version spec.
```suggestion
    - decorator >=3.4.2
```